mage/k8s/lib.go

```diff
@@ -266,6 +266,21 @@ func Env() {
 func InstallTool() {
 	loadClusterSettings()
 	clusterActions.InstallTool()
+	installHelm()
+}
+
+func installHelm() {
```

Comment:
I think that this function should be moved into `mage/helm/lib.go` that already exists